src/App.jsx

```diff
@@ -3,9 +3,14 @@ import './App.css'
 function App() {
 
   return (
-    <h1>
-      Hello Slava!
-    </h1>
+    <header>
```

Comment:
Man, you have to create a reusable component. If you just copy all your html markup into App.jsx it doesn't make much sense. 

So please create a `Search.jsx` inside of `components` folder and create all the logic and styles there.
Your component should have reasonable props.

And try to think what else might be extracted for a separate component. Imagine this project doesn't stop with one page. And we would wanna use the same elements multiple times in different places.